p2p_wallet/Scenes/Main/BankTransfer/StrigaRegistration/FirstStep/StrigaRegistrationFirstStepViewModel.swift

```diff
@@ -76,7 +76,7 @@ final class StrigaRegistrationFirstStepViewModel: BaseViewModel, ObservableObjec
             .map { $0.split(separator: ".") }
             .map { components in
                 if components.count == Constants.dateFormat.split(separator: ".").count {
-                    return StrigaUserDetailsResponse.DateOfBirth(year: Int(components[2]), month: Int(components[1]), day: Int(components[0]))
+                    return StrigaUserDetailsResponse.DateOfBirth(year: String(components[2]), month: String(components[1]), day: String(components[0]))
```

Comment:
I would recommend have a method to convert str to date components, move it to helper, so it would be easy to add localizations and some modifiers like .stripCharacters.. etc